build-parent/pom.xml

```diff
@@ -112,6 +112,86 @@
 
   <build>
     <plugins>
+      <plugin>
```

Comment:
The `failsafe` & `surefire` plugin present in the root POM were being used by `build-parent` & `testsuite-parent`. Since I was not able to figure out (in my limited weekend time) the problem with why `testsuite-parent` cannot be brought under `build-parent`, I am duplicating the plugin configurations in these two files for now. Will take care of fixing it in a subsequent PR.